ivy/array/linear_algebra.py

```diff
@@ -94,6 +94,33 @@ def matrix_rank(
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
         return ivy.matrix_rank(self._data, rtol, out=out)
+        """
+        ivy.Array instance method variant of ivy.matrix_rank. This method returns 
+        the rank (i.e., number of non-zero singular values) of a matrix (or a stack of
+        matrices).
+        
```

Comment:
Hi @hyeonukim 
Can you add the parameters and return docstrings as well to complete this PR? You can follow [this](https://github.com/unifyai/ivy/blob/f6738a28ca5d997f70ef6a8e4be7d367d35bf32b/ivy/array/linear_algebra.py#L31)